c++/src/capnp/schema.capnp

```diff
@@ -192,9 +192,16 @@ struct Node {
     struct Member {
       docComment @0 :Text;
       # Doc comment on the member.
+      bytePosition @1 :BytePosition;
+      brandBytePositions @2 :List(BytePosition);
```

Comment:
@kentonv 

I added the byte position of brand parameters. Could you tell me how to set parameters?

Reply:
I might have misunderstand.  
I was considering supporting the following case.
```
struct Map(Key, Value) {
  entries @0 :List(Entry);
  struct Entry {
    key @0 :Key;
    value @1 :Value;
  }
}

struct People {
  byName @0 :Map(Text, Person);
  # Maps names to Person instances.
}
```

Anyway, with your schema proposal, this can also be resolved, so there is no problem.

Reply:
Ah yes, but what if you have:

```
someField @0 :SomeType(Foo, Bar(Baz(Qux), Corge(Grault, Garply)))
```

As you can see, it's not just a simple list of brand parameters. The type can be a whole expression tree. So I think the separate "identifiers list" approach is what we need here.